README.md

```diff
@@ -25,13 +25,17 @@ Generates bindings file `path/to/definitions.cs`
 
 To integrate the bindings into your projects, simply add the generated bindings file to your project.
 There are a couple of requirements to compile the generated bindings file:
-- `dotnet` version `6.0` or higher
+- .NET core `6.0` or higher
```

Comment:
As of today, it seems the accepted term is just `.NET` or `dotnet`
https://learn.microsoft.com/en-us/dotnet/fundamentals/implementations